### PR TITLE
Star fix, refactor incoming list, photo upload

### PR DESCRIPTION
**Pull request by positivefx**

### CODE REVIEW

File: client/src/components/RatingReviews/RatingReviews.jsx [hunk 1]
```diff
@@ -11,11 +11,12 @@ function RatingReviews() {
   const { productId } = useContext(ProductContext);
 
   const [sortView, setSortView] = useState('relevant');
-  const [sortedList, setSortedList] = useState(sortedList);
+  const [sortedList, setSortedList] = useState(null);
```
Comment (addisonhernandez) [inline client/src/components/RatingReviews/RatingReviews.jsx:14]: :+1:

File: client/src/components/RatingReviews/RatingReviews.jsx [hunk 2]
```diff
@@ -76,21 +75,15 @@ function RatingReviews() {
...
+      {sortedList && (
+        <ReviewList sortedList={sortedList} sortStarFilter={sortStarFilter} />
+      )}
 
       <div className="RR-write-review">
         <button
           className="RR-write-review-button"
           type="button"
-          onClick={handleWriteReviewClick}
+          onClick={() => setShowWriteReview(true)}
```
Comment (addisonhernandez) [inline client/src/components/RatingReviews/RatingReviews.jsx:86]: good catch

File: client/src/components/RatingReviews/Reviews/ReviewList.jsx [hunk 3]
```diff
@@ -1,13 +1,33 @@
...
+
   return (
-    <div className='RR-review-list'>
+    <div className="RR-review-list">
       <h4>Review List </h4>
-      {sortedList.map((eachReview) => (
-        <ReviewEntry key={eachReview.review_id} eachReview={eachReview} />
-      ))}
+      {sortedList
+        .slice(0, showMoreReview)
```
Comment (addisonhernandez) [inline client/src/components/RatingReviews/Reviews/ReviewList.jsx:16]: I think you need to filter _before_ slicing

File: client/src/components/RatingReviews/WriteReview/WriteReview.jsx [hunk 4]
```diff
@@ -1,8 +1,9 @@
 import React, { useState } from 'react';
 import axios from 'axios';
+import { escapeValue } from 'Utilities';
```
Comment (addisonhernandez) [inline client/src/components/RatingReviews/WriteReview/WriteReview.jsx:3]: :fire:

**Review by addisonhernandez — APPROVED** [review]

File: client/src/components/RatingReviews/Reviews/ReviewList.jsx [hunk 5]
```diff
@@ -13,11 +13,11 @@ function ReviewList({ sortedList, sortStarFilter }) {
     <div className="RR-review-list">
       <h4>Review List </h4>
       {sortedList
-        .slice(0, showMoreReview)
         .filter(
           (eachReviewObj) =>
             sortStarFilter === 0 || eachReviewObj.rating === sortStarFilter
         )
+        .slice(0, showMoreReview)
```
Comment (addisonhernandez) [inline client/src/components/RatingReviews/Reviews/ReviewList.jsx:20]: :rocket:

**Review by addisonhernandez — APPROVED** [review]

**Comment by Acid-Override** [comment]
[Wed, Apr 8 - Daily Merge Party 🎉](https://trello.com/c/AfafjKbZ/40-wed-apr-8-daily-merge-party-%F0%9F%8E%89)

**Pull request merged**